src/main/python/org/broadinstitute/hellbender/tools/gcnv/gcnvkernel/structs/metadata.py

```diff
@@ -265,21 +266,22 @@ def read_sample_coverage_metadata(self, input_file: str) -> List[str]:
             contig_list = []
             for row in reader:
                 row_num += 1
+                num_header_elems = None
```

Comment:
I think this line breaks things.

Reply:
huh, I had fixed this earlier... weirdness.